container-guide/adoc/containers-bci.adoc

```diff
@@ -322,6 +322,21 @@ buildah bud --layers --secret=id=SCCcredentials,src=/path/to/SCCcredentials \
     --secret=id=SUSEConnect,src=/path/to/SUSEConnect .
 ----
 
+[NOTE]
+====
+container-suseconnect runs automatically every time you invoke {zypper}. If you are not using a registered {slsa} host, you may see the following error message:
+
+....
+> zypper ref
+Refreshing service 'container-suseconnect-zypp'.
+Problem retrieving the repository index file for service 'container-suseconnect-zypp':
+[container-suseconnect-zypp|file:/usr/lib/zypp/plugins/services/container-suseconnect-zypp] 
+Warning: Skipping service 'container-suseconnect-zypp' because of the above error.
+....
```

Comment:
Will this render correctly without the `----` blocks?

Reply:
Do you mean `....`?

Reply:
```suggestion
[literal]
....
> zypper ref
Refreshing service 'container-suseconnect-zypp'.
Problem retrieving the repository index file for service 'container-suseconnect-zypp':
[container-suseconnect-zypp|file:/usr/lib/zypp/plugins/services/container-suseconnect-zypp] 
Warning: Skipping service 'container-suseconnect-zypp' because of the above error.
....
```